ghost/portal/src/components/Frame.styles.js

```diff
@@ -662,7 +662,7 @@ const FrameStyles = `
     letter-spacing: -.1px;
     font-weight: 700;
     text-transform: uppercase;
-    color: #15212a;
+    color: var(--grey0);
```

Comment:
This logo is marketing material which should be the same everywhere. These values should remain hardcoded.